clients/TypeScript/packages/client/test/StateQuery.test.ts

```diff
@@ -30,7 +30,7 @@ describe('Local state queries', () => {
           await client.release()
         }
       } catch (error) {
-        expect(error.code).toBe('EAI_AGAIN')
+        expect(error.code).toMatch(/EAI_AGAIN|ENOTFOUND/)
```

Comment:
Are you running MacOs :thinking: ? I recall that socket I/O errors and file I/O are quite inconsistent across platforms ... (especially Windows :grimacing: ...). I kept the EAI_AGAIN in here, but I never encountered it on my linux machine.

Reply:
My main dev machine is linux, but occasionally I'm on macOS. Will cover this off in #59, and have created #60 for the server.